amr-wind/physics/TerrainDrag.cpp

```diff
@@ -0,0 +1,180 @@
+#include "amr-wind/physics/TerrainDrag.H"
+#include "amr-wind/CFDSim.H"
+#include "AMReX_iMultiFab.H"
+#include "AMReX_MultiFabUtil.H"
+#include "AMReX_ParmParse.H"
+#include "AMReX_ParReduce.H"
+#include "amr-wind/utilities/trig_ops.H"
+#include "amr-wind/utilities/IOManager.H"
+
+namespace amr_wind::terraindrag {
+
+namespace {} // namespace
+
+TerrainDrag::TerrainDrag(CFDSim& sim)
+    : m_sim(sim)
+    , m_repo(sim.repo())
+    , m_mesh(sim.mesh())
+    , m_velocity(sim.repo().get_field("velocity"))
+    , m_terrainBlank(sim.repo().declare_field("terrainBlank", 1, 1, 1))
+    , m_terrainDrag(sim.repo().declare_field("terrainDrag", 1, 1, 1))
+    , m_terrainz0(sim.repo().declare_field("terrainz0", 1, 1, 1))
+{
+    std::string terrainfile("terrain.amrwind");
```

Comment:
What's the format for these files? Are they expected to have headers for the labels where you need to skip the first line?